app/helpers/public_document_routes_helper.rb

```diff
@@ -15,6 +15,8 @@ def preview_document_path(edition, options = {})
   end
 
   def document_url(edition, options = {}, _builder_options = {})
+    return edition.link if edition.is_a? RummagerDocumentPresenter
```

Comment:
Similar to my comment on `document_list_from_rummager.html.erb`

Reply:
Thanks, fixed